### PR TITLE
adding rfc for webview information

### PR DESCRIPTION
**Pull request by chrisdavidmills**
> What was wrong/why is this fix needed? (quick summary only)

In the previous browser compat data project meeting, we decided that it would be good to add information to MDN on dealing with WebView environments, from the perspective of a web developer trying to support them in their apps/sites. This RFC aims to define that work.

> MDN URL of main page changed

n/a

> Issue number (if there is an associated issue)

n/a

> Anything else that could help us review it

n/a

### CODE REVIEW

File: rfcs/webview-information.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# MDN content project: ARIA roles reference docs
+
+This RFC proposes that we work on adding WebView documentation to MDN, to help
+web developers figure out how test and support WebView environments (i.e. iOS
```
Comment (sideshowbarker) [inline rfcs/webview-information.md:4]:
```suggestion
web developers figure out how to test and support WebView environments (i.e. iOS
```

File: rfcs/webview-information.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+and Android WebView-based "mega app" type environments).
+
+This would include:
+
+- What is a WebView envrionment?
```
Comment (sideshowbarker) [inline rfcs/webview-information.md:9]:
```suggestion
- What is a WebView environment?
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
Thanks @sideshowbarker ; suggestions committed.

File: rfcs/webview-information.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+web developers figure out how to test and support WebView environments (i.e. iOS
+and Android WebView-based "mega app" type environments).
+
+This would include:
+
+- What is a WebView environment?
+- How frequently WebView environments are used and in what regions they are
+  used the most.
+- How WebView environments differ from their browser counterparts.
+- How to test WebView environments.
```
Comment (andreban) [inline rfcs/webview-information.md:13]: It this more about "How do I test if my site is running in a WebView environment at runtime?" or "How do I test my site in a WebView environment?"

Comment (chrisdavidmills) [reply rfcs/webview-information.md:13]: I agree that we need both; I'll update the RFC to include this. Thanks!

**Comment by chrisdavidmills** [comment]
I think this is looking good now; merging.

**Comment by tomayac** [comment]
@chrisdavidmills I have done some [research on WebViews](https://research.google/pubs/pub46739/) in the past. The content of this paper is up for grabs and freely available (licensed as CC BY-NC-ND 4.0).

(Meta: I found the [RFC](https://github.com/mdn/content/blob/main/rfcs/webview-information.md), but don't really know how to [submit feedback](https://github.com/mdn/content/tree/main/rfcs#getting-feedback), which is probably simply due to the fact that the RFC hasn't been _"sent around to key stakeholders"_.)

**Comment by chrisdavidmills** [comment]
> @chrisdavidmills I have done some [research on WebViews](https://research.google/pubs/pub46739/) in the past. The content of this paper is up for grabs and freely available (licensed as CC BY-NC-ND 4.0).

Thanks @tomayac ; that's really useful, and I'll add it to the RFC.

> (Meta: I found the [RFC](https://github.com/mdn/content/blob/main/rfcs/webview-information.md), but don't really know how to [submit feedback](https://github.com/mdn/content/tree/main/rfcs#getting-feedback), which is probably simply due to the fact that the RFC hasn't been _"sent around to key stakeholders"_.)

The idea is that it can be a living document and be commented on and updated after it is published. We didn't really think about the best way to give feedback after publication, but I think any suitable way is fine, be it a google doc, or issue, or PR even...

**Pull request merged**